clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -387,6 +387,11 @@ Changes in existing checks
   <clang-tidy/checks/readability/redundant-inline-specifier>` check to properly
   emit warnings for static data member with an in-class initializer.
 
+- Improved :doc:`readability-redundant-member-init
+  <clang-tidy/checks/readability/redundant-member-init>` check to avoid
+  false-positives when type of the member does not match type of the
+  initializer.
+
```

Comment:
I think this note should mention that base-class initializers are now checked as well. It's a minor user-facing change.

(While you're at it: the last to `type` need a `the` beforehand)

Reply:
Nothing has changed for base-class initializers.